fbuapp/View Controllers/HomeViewController.m

```diff
@@ -44,21 +50,22 @@ - (void)viewDidLoad {
         make.edges.equalTo(self.view).with.insets(UIEdgeInsetsMake(0, 0, 0, 0));
     }];
     
-    CategoryHeaderView *pillSelector = [[CategoryHeaderView alloc] initWithFrame:CGRectMake(0,0,self.view.frame.size.width,60)];
-    pillSelector.delegate = self;
-    self.feed.tableHeaderView = pillSelector;
+    self.pillSelector = [[CategoryHeaderView alloc] initWithFrame:CGRectMake(0,0,self.view.frame.size.width,60)];
+    self.pillSelector.delegate = self;
+    self.feed.tableHeaderView = self.pillSelector;
     
     self.navigationItem.rightBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:@"Filter" style:UIBarButtonItemStylePlain target:self action:@selector(presentFilterViewController:)];
 }
 
-- (IBAction)presentFilterViewController:(id)sender {
+- (IBAction)presentFilterViewController:(id)sender{
     FilterViewController *filterVCObj =[[FilterViewController alloc] initWithNibName:@"FilterViewController" bundle:nil];
     filterVCObj.currentLocation = [PFGeoPoint geoPointWithLocation: self.feed.currentLocation];
+    filterVCObj.savedQuery = self.savedQuery;
     [self presentViewController:[[UINavigationController alloc] initWithRootViewController:filterVCObj] animated:YES completion:nil];
     filterVCObj.delegate = self;
 }
 
-- (void) fetchPosts {
+- (void)fetchPosts{
```

Comment:
Not sure if it's just my view, but there seems to be a chunk of info missing. Such as extra starter brackets and an if statement.

Reply:
oh yeah i think it skipped lines 72 - 79